.pre-commit-config.yaml

```diff
@@ -1,7 +1,11 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
--   repo: https://github.com/franzpoeschel/openPMD-api-pre-commit-hooks
-    rev: aa38b2ad4fae42db05c19de211f43df6a7981bc5
+-   repo: local
```

Comment:
We could use https://github.com/pre-commit/mirrors-clang-format for this :)

Reply:
I like the current solution actually because it gives us a script that users can also call "bare-metal" if they want, without having to set up pre-commit. Also, it's a 5-line script that saves us from relying on an external dependency.
I can move it to the workflows thing, sure.

Reply:
Jup, I think that's a good point and it integrates well.